dts/bindings/net/wireless/silabs,siwx91x-nwp.yaml

```diff
@@ -31,3 +31,51 @@ properties:
       - deep-sleep-without-ram-retention
       - deep-sleep-with-ram-retention
     required: true
+
+  rtc-from-host:
+    type: boolean
+    description: |
+      Enables Real-Time Clock (RTC) synchronization from the host system.
+      When enabled, the NWP uses the host's RTC for timekeeping and certificate
+      validation timing.
+      Enable this when the host MCU provides a valid RTC source.
+
+  support-1p8v:
+    type: boolean
+    description: |
+      Enables 1.8 V operation support for the NWP power domain.
+      Enable this if the board powers the NWP with 1.8 V instead of 3.3 V.
+
+  disable-xtal-correction:
+    type: boolean
+    description: |
+      Disables automatic 40 MHz XTAL frequency correction.
+      Enable this for calibration applications, or when using a custom XTAL
+      (part number other than 8Y40070013) to avoid automatic compensation.
+
+  nwp-qspi-80mhz-clk:
+    type: boolean
+    description: |
+      Enables 80 MHz QSPI clock mode for the NWP flash interface to improve
+      throughput. Enable this if the external flash device supports 80 MHz
+      operation; otherwise, keep it disabled.
```

Comment:
All the attributes are related to NWP. You don't need to prefix it with `nwp-`.

Note for readers: the Zephyr model would require to have a clock-management driver for this feature. However, the network co-processor (NWP) is not (yet) fully compatible with that model. I suggest to keep this attribute for now.

(@ArunmaniAlagarsamy2710, you may want to add this comment somewhere in the binding)